features/netsocket/emac-drivers/TARGET_RDA_EMAC/RdaWiFiInterface.cpp

```diff
@@ -0,0 +1,251 @@
+/* LWIP implementation of NetworkInterfaceAPI
```

Comment:
Don't copy&paste license header sections from other files.

I don't think you can add `Copyright (c) 2019 ARM Limited` if you are not working in Arm. Perhaps your employer?